### ISSUE TITLE
Expand Query Graph node (without curie) based on BioLink model hierarchy

### ISSUE THREAD

**Issue by kevinxin90** [biothings/biothings_explorer#102]
See details in this issue: https://github.com/NCATSTranslator/testing/issues/12

**Comment by kevinxin90** [comment]
Fixed in: b123cf5bcbff40670254bf48d2bb4ca362f42ab8

**Comment by andrewsu** [comment]
Adding a brief comment because I keep forgetting this point.  I normally would have expected that if I had a query that asked for a `biolink:Gene` (example below), then replacing that would `biolink:GeneOrGeneProduct` would produce a superset of those results.  In fact, `biolink:GeneOrGeneProduct` is _not_ a parent of `biolink:Gene`, but rather it is a mixin.  `biolink:GenomicEntity` is the parent of `biolink:Gene`, and as expected, querying with that instead of `biolink:Gene` does produce a superset of results (including Genes, but also Variants)

```
{
    "message": {
        "query_graph": {
            "edges": {
                "e01": {
                    "subject": "n1",
                    "object": "n0"
                }
            },
            "nodes": {
                "n0": {
                    "id": "MONDO:0008753",
                    "category": "biolink:Disease"
                },
                "n1": {
                    "category": "biolink:Gene"
                }
            }
        }
    }
}
```
